src/services/CatalogService.ts

```diff
@@ -31,4 +44,60 @@ export const catalogService: ICatalogService = {
 
     await events.createEvent(eserviceSeedToCreateEvent(eserviceSeed));
   },
+  updateEService: async (
+    eServiceId: string,
+    eservicesSeed: ApiEServiceSeed
```

Comment:
Shouldn't this be a different `UpdateEServiceSeed` in the spec? without `producerId`? Is this accepted here because it's a subtype?

Reply:
In the open API specs, it seems identical

Reply:
ah, my mistake sorry... `producerId` is in the responses